packages/ripple-nuxt-tide/lib/core/middleware.js

```diff
@@ -189,12 +189,18 @@ export default async function (context, results) {
           }
         }
         // site section nav
-        if (results.tidePage.field_show_site_section_nav) {
-          const addSectionNavMenu = await context.app.$tide.getSiteData(results.tidePage.section).then(appSectionData => {
+        if (results.tidePage.section) {
+          const addSectionNavMenu = await context.app.$tide.getSiteData(results.tidePage.section).then(async siteData => {
+            // save alerts if site section has them
+            if (context.app.$tide.isModuleEnabled('alert')) {
```

Comment:
I would suggest update here a little but it's ok to pass based on the time pressure.
The original code(from Guy) is to `addSectionNavMenu`, so insert `alert` related logic into it is not right for me. We can get the sitedata first then do the two logic work after that.